tsconfig.base.json

```diff
@@ -15,14 +15,17 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
-      "@elewa-website/elements/layout/header": [
-        "libs/elements/layout/header/src/index.ts"
-      ],
```

Comment:
This removes the header aliased path from the codebase

Reply:
@LemmyMwaura this path is still there it was not removed

Reply:
Okay, I've seen it.